integration-test/976-fractional-pois.py

```diff
@@ -12,15 +12,15 @@
     'name': 'New Jersey - Pennsylvania' })
 
 assert_has_feature(
-    5, 9, 12, 'roads',
-    { 'min_zoom': 3.0 , 'id': int, 'sort_key': 383,
+    7, 37, 48, 'roads',
+    { 'min_zoom': 3 , 'id': int, 'shield_text': '95',
     'source': 'naturalearthdata.com' })
 
 # There is no transit data from Natural Earth
 
 assert_has_feature(
-    5, 9, 12, 'water',
-    { 'min_zoom': 0 , 'id': 1144,
```

Comment:
Zoomed "in" again, so that the data comes from `ne_10m_lakes` rather than one of the other, coarser NE tables.